### PR TITLE
Added Json files

### LINKED ISSUE #20: Add mock data as JSON files
**Describe the bug**
The mock data has to be available as JSON files in the public directory.

**To Reproduce**
The public directory does not have mock data as JSON files. 

**Expected behavior**
Each UI screen has hard coded data. 
The data has to come from mock data files and files would be in JSON format. 
The files need to reside in  directory named public  of the repo.

### PR DESCRIPTION
**Pull request by Korra15**
Fixes   #20

### CODE REVIEW

File: public/jsonFiles/users/userProfile.json [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    ],
+    "role" : [
+        {
+            "id" : "01",
+            "role_name" : "role1",
+            "role_description" : " "
+        }
+    ],
+    "skill_role" : [
+        {
```
Comment (aprajshekhar) [inline public/jsonFiles/users/userProfile.json:27]: This is a many-to-many mapping table. Hence, not required for UI

File: public/jsonFiles/users/userProfile.json [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+        }
+    ],
+    "source" : [
+        {
+            "id" : "1",
+            "source_name" : "source1"
+        }
+    ],
+    "source_skills" : [
+        {
```
Comment (aprajshekhar) [inline public/jsonFiles/users/userProfile.json:40]: This is a many-to-many mapping table. Hence, not required for UI

**Comment by hybridx** [comment]
hi @Korra15 can you please rebase, maybe that'll fix the lint errors

**Comment by Korra15** [comment]
> hi @Korra15 can you please rebase, maybe that'll fix the lint errors

Thank You sir.

**Pull request closed without merging**